miden-tx/src/testing/tx_context.rs

```diff
@@ -125,25 +162,9 @@ impl TransactionContextBuilder {
         }
     }
 
-    pub fn with_acc_type(account_type: MockAccountType) -> Self {
```

Comment:
Removed the `MockAccountType` in favor of methods with explicit parameters. This removed duplicate (StandarNew and StandardExisting can be implemented by the same code). Removed some bad naming (`empty_reserved_slot` for FungibleFaucet was used to initialize its balance, the boolean name was unrelated).